app/views/layouts/_flash.html.erb

```diff
@@ -5,7 +5,12 @@
         <% if key == "error" %>
           <h3 class="govuk-<%= key %>-summary__title" id="<%= key %>-summary-heading" data-error-message="<%= value %>"><%= value %></h3>
         <% else %>
-          <h3 class="govuk-<%= key %>-summary__title" id="<%= key %>-summary-heading"><%= value %></h3>
+          <% if value.is_a?(Hash) %>
```

Comment:
Typechecking in a dynamic language generally feels like an anti-pattern to me, could this warrant a separate template for errors we might have manually set rather than forwarded from the backend? Or is it the case that the backend sometimes provides data in this format?

Reply:
These are flash messages that we manually create by setting them in `flash` (The in built Rails thing). Typically we only support a single string, and without updating them all to hashes I added this here. Not sure how to add this functionality  

Wonder if there's a nicer approach though :thinking: given I'll try find a way of adding this

Reply:
maybe 
``` ruby
if (value.dig(:title).present? || value.dig(body).present?)
```

or change the `key` to something else